src/patternfly/components/Nav/nav.scss

```diff
@@ -558,15 +563,11 @@
   margin-left: auto;
   font-size: var(--pf-c-nav__toggle--FontSize);
   line-height: 1;
-}
 
-// Toggle icon
-.pf-c-nav__toggle-icon {
-  display: inline-block;
-  transition: var(--pf-c-nav__toggle-icon--Transition);
-
-  .pf-c-nav__item.pf-m-expanded & {
-    transform: var(--pf-c-nav__item--m-expanded__toggle-icon--Transform);
+  > * {
```

Comment:
is it possible for the toggle wrapper to rotate and not the icon inside since we want to avoid styling the child element